### DISCUSSION TITLE
How to make this one overflow the screen like before?

### DISCUSSION THREAD

**Discussion by nexus9xx** [PoomSmart/YTABConfig#23] [Q&A]
The new interface is great but this one is annoying...How to make this one overflow the screen like before?  it looks so tiny now... 

![F16E19B9-6ECC-4506-A21F-EE094D4F9966](https://user-images.githubusercontent.com/102811744/198834498-078874af-d589-4f11-9dcd-d0b952ca3f67.jpeg)
![AAC14E01-550D-425C-AA49-5C6B0810B1C5](https://user-images.githubusercontent.com/102811744/198834502-d7b44883-3135-4314-b1a4-652a96d4c5e8.jpeg)

**Comment by lemonke444** [comment]
try deactivating this flags YTColdConfig.uiSystemsClientGlobalConfigEnableModernButtonsForNative: 0
YTColdConfig.uiSystemsClientGlobalConfigIosEnableSnackbarModernization: 0

**Reply by nexus9xx** [reply]
I have tried it and no change...

**Reply by lemonke444** [reply]
Try deactivating this flags :\

uiSystemsClientGlobalConfigUseDarkerPal etteBgColorForNative 
uiSystemsClientGlobalConfigUseDarkerPal etteTextColorForNative 
useDarkerPaletteBgColorForElements
useDarkerPaletteTextColorForElements

**Reply by nexus9xx** [reply]
Oh.. you seem to be confused with the color of the bar... I mean to bring it back to its original state, not as small as it is now :/

**Reply by lemonke444** [reply]
im sorry :(

**Reply by lemonke444** [reply]
i found a solution just downgrade the app. im using youtube version  16.42.3 and has the old interface :)
